packages/renderer/src/lib/statusbar/StatusBarItem.svelte

```diff
@@ -1,3 +1,20 @@
+<style>
+.status-bar-item {
+  color: white;
+  position: relative;
+  display: inline-block;
+}
+
+.status-bar-item .badge {
+  position: absolute;
+  top: -4px;
+  right: -4px;
+  padding: 4px 4px;
+  border-radius: 50%;
+  background-color: var(--pf-global--primary-color--100);
```

Comment:
Is there any way to not have the hardcoded style / CSS here? What do you think @benoitf 

Otherwise, this PR LGTM.

Reply:
I updated, and use the class when possible!